Formula/libevent.rb

```diff
@@ -11,22 +11,32 @@ class Libevent < Formula
     sha256 "ef703db1b4cbdab35b89aabe80c225dd9b7a2c3ea14b1eae681478c5b9df15fe" => :yosemite
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  head do
+    url "https://github.com/libevent/libevent", :using => :git
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+  end
+
   depends_on "doxygen" => :build
-  depends_on "libtool" => :build
   depends_on "pkg-config" => :build
-  depends_on "openssl"
+  depends_on "openssl" => :recommended
```

Comment:
the default behaviour will be the same (pulling openssl and linking with it), this option only allows for the formula to skip building against openssl.

it is important to note that at least in Sierra, there is no risk of linking against the system openssl because the headers are missing and build will fail.